app/components/Documentation/EditDocumentation.tsx

```diff
@@ -229,6 +293,78 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
                 )}
               </>
             )}
+            <div
```

Comment:
move all pagination to a separate component

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9